<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dCreate_user</class>
 <widget class="QDialog" name="dCreate_user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создать пользователя</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>400</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Логин</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_select">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>SELECT</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_insert">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>INSERT</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_update">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>UPDATE</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_delete">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>83</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_delete_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>83</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_insert_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>INSERT</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_select_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>SELECT</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_update_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>UPDATE</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_delete_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>83</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_insert_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>INSERT</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_select_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>SELECT</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_update_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>UPDATE</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_delete_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>83</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_insert_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>INSERT</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_select_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>SELECT</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_update_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>UPDATE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сделки </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Продукты</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Поставки</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Поставщик</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dCreate_user</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dCreate_user</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
